### PR TITLE
Add running data studios default limit

### PR DESCRIPTION
**Pull request by jason-seqera**
- https://seqera.atlassian.net/browse/EDU-340
- Preview: https://deploy-preview-232--seqera-docs.netlify.app/platform/24.1/limits#data-studio

### CODE REVIEW

File: platform_versioned_docs/version-24.1/limits/limits.mdx [hunk 1]
```diff
@@ -37,3 +37,9 @@ Some Enterprise instances on older licenses are limited to 100 labels per worksp
 | Versions per dataset | 100   |
 
 If you need higher limits and capabilities, [contact us](https://seqera.io/contact-us/) to discuss your application requirements.
+
+### Data Studio
```
Comment (justinegeffen) [inline platform_versioned_docs/version-24.1/limits/limits.mdx:41]:
```suggestion
### Data Studios
```

File: platform_versioned_docs/version-24.1/limits/limits.mdx [hunk 2]
```diff
@@ -37,3 +37,9 @@ Some Enterprise instances on older licenses are limited to 100 labels per worksp
...
+
+| Description          | Value |
+| -------------------- | ----- |
+| Running data studios | 100   |
```
Comment (justinegeffen) [inline platform_versioned_docs/version-24.1/limits/limits.mdx:45]:
```suggestion
| Running Data Studio sessions | 100   |
```

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
@jason-seqera, this looks good to me. Thank you!

**Review by robnewman — APPROVED** [review]
Approved, but now I've noticed that we don't display ANY free-tier quotas in this "Limits" page. As part of the Free Tier Messaging PRD we should update this page to note the free-tier limits clearly.

**Pull request merged**